installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -7,6 +7,11 @@ You can install {product} in an air-gapped environment to ensure security and me
 
 To install {product-short} in an air-gapped environment, you must have access to the `registry.redhat.io` and the registry for the air-gapped environment.
 
+[NOte]
+====
+If you want to configure and use the telemetry data collection feature in an air-gapped environment, see the link:{LinkAdminGuide}#assembly-rhdh-telemetry_admin-rhdh[Telemetry data collection] section. 
```

Comment:
```suggestion 
```
telemetry is usually not used in air-gapped environments as the nature of air-gap environment makes in impossible to send data out which would defeat the pursue of air-gap